packages/devops/scripts/ci/testInternalDependencies.sh

```diff
@@ -2,7 +2,7 @@
 DIR=$(dirname "$0")
 for PACKAGE in $(${DIR}/../../../../scripts/bash/getInternalDependencies.sh); do
     # skip the following packages - they get tested separately as they require db access
-    if [[ "$PACKAGE" == "database" || "$PACKAGE" == "data-api" || "$PACKAGE" == "auth" || "$PACKAGE" == "indicators" ]]; then
+    if [[ "$PACKAGE" == "database" || "$PACKAGE" == "data-api" || "$PACKAGE" == "auth" || "$PACKAGE" == "indicators" || "$PACKAGE" == "server-boilerplate" ]]; then
```

Comment:
Do `server-boilerplate` tests require db access? I think it's fine if it's not included in this list?

Reply:
It depends on the test but yeah the package does have db access to it would be a probable yes.